src/style.scss

```diff
@@ -186,6 +200,6 @@ svg {
   --auro-size-lg: 1rem;
 
   position: relative;
-  top: 2px;
+  top: 4px;
```

Comment:
Do we still need this change?

Reply:
Yes... there is an issue with the icon that I am quickly fixing.